frontend/src/pages/profile/profile.jsx

```diff
@@ -37,22 +49,32 @@ class Profile extends React.Component {
             skills: form.state.user.skills.split(' ')
         };
 
-        axios.put(process.env.REACT_APP_BACKEND_URL + '/user', {old: oldUser, new: newUser}).then(res => {
-            this.props.login(res.data.user || {});
-        });
+        // update user document 
+        modifyUser(this.state.user, newUser).then((success) => {
+            if(success) { 
+                console.log('updated user')
+                this.props.login(newUser || {})
+            }
+        })
     }
     
     deletePressed = () => { 
```

Comment:
does this function do anything / has it been replaced by a firebase call? If its been replaced and isnt used then get rid of it, but if it hasn't been replaced then just leave the call to our backend there for now